scripts/eosio_build_oraclelinux.sh

```diff
@@ -0,0 +1,36 @@
+echo "OS name: ${NAME}"
+echo "OS Version: ${VERSION_ID}"
+echo "CPU cores: ${CPU_CORES}"
+echo "Physical Memory: ${MEM_GIG}G"
+echo "Disk space total: ${DISK_TOTAL}G"
+echo "Disk space available: ${DISK_AVAIL}G"
+
+( [[ $NAME == "Oracle Linux Server" ]] && [[ "$(echo ${VERSION} | sed 's/ .*//g')" < 8 ]] ) && echo " - You must be running Oracle Linux 8 or higher to install EOSIO." && exit 1
```

Comment:
We should do this comparison in a mathematical context (`(( ))`) instead of normal `test` (`[[ ]]`) because, otherwise, this will fail once we get to 10.

<img width="390" alt="Screen Shot 2021-09-29 at 2 08 28 PM" src="https://user-images.githubusercontent.com/34947245/135324986-9fab78fd-466f-424e-964a-47e4e470258f.png">